libs/community/langchain_community/llms/bedrock.py

```diff
@@ -93,11 +107,26 @@ class LLMInputOutputAdapter:
 
     @classmethod
     def prepare_input(
-        cls, provider: str, prompt: str, model_kwargs: Dict[str, Any]
+        cls,
+        provider: str,
+        model_kwargs: Dict[str, Any],
+        prompt: Optional[str] = None,
+        system: Optional[str] = None,
+        messages: Optional[List[Dict]] = None,
     ) -> Dict[str, Any]:
         input_body = {**model_kwargs}
         if provider == "anthropic":
-            input_body["prompt"] = _human_assistant_format(prompt)
+            if messages:
+                input_body["anthropic_version"] = "bedrock-2023-05-31"
+                input_body["messages"] = messages
+                if system:
+                    input_body["system"] = system
+                if "max_tokens" not in input_body:
+                    input_body["max_tokens"] = 1024
```

Comment:
where is this default value of 1024 for max_tokens coming from?

Reply:
This was just part of the testing I was doing. If Anthropic has any defaults, we can use them here, I also noticed the default in Bedrock console for sonnet is 2000, but lower for other models.

Reply:
1024 as a default matches what we switched the anthropic connectors too.